test/erlfmt_SUITE.erl

```diff
@@ -1106,3 +1135,95 @@ contains_pragma(Config) when is_list(Config) ->
             "-module(pragma).\n"
         )
     ).
+
+insert_pragma(Config) when is_list(Config) ->
+    ?assertEqual(
+            "%% @format\n"
+            "\n"
+            "-module(pragma).\n",
+        insert_pragma_string(
+            "-module(pragma).\n"
+        )
+    ),
+    ?assertEqual(
+            "%% @format\n"
+            "\n"
+            "-module(pragma).\n"
+            "\n"
+            "-export([f/3]).\n"
+            "\n"
+            "f(_Arg1, _Arg2, _Arg3) ->\n"
+            "    ok.\n",
+        insert_pragma_string(
+            "-module(pragma).\n"
+            "\n"
+            "-export([f/3]).\n"
+            "\n"
+            "f(_Arg1,_Arg2,   _Arg3) ->\n"
+            "ok.\n"
+        )
+    ),
+    ?assertEqual(
+            "%% attached comment\n"
+            "%% @format\n"
+            "-module(pragma).\n"
+            "\n"
+            "-export([f/3]).\n",
+        insert_pragma_string(
+            "%% attached comment\n"
+            "-module(pragma)\n."
+            "\n"
+            "-export([f/3]).\n"
+        )
+    ),
+    ?assertEqual(
+            "%% single comment\n"
+            "%% @format\n"
+            "\n"
+            "-module(pragma).\n"
+            "\n"
+            "-export([f/3]).\n",
+        insert_pragma_string(
+            "%% single comment\n"
+            "\n"
+            "-module(pragma)\n."
+            "\n"
+            "-export([f/3]).\n"
+        )
+    ),
+    ?assertEqual(
+            "%% LICENSE\n"
+            "%% LICENSE\n"
+            "%% LICENSE\n"
+            "%% LICENSE\n"
+            "%% @format\n"
+            "\n"
+            "-module(pragma).\n"
+            "\n"
+            "-export([f/3]).\n",
+        insert_pragma_string(
+            "%% LICENSE\n"
+            "%% LICENSE\n"
+            "%% LICENSE\n"
+            "%% LICENSE\n"
+            "\n"
+            "-module(pragma)\n."
+            "\n"
+            "-export([f/3]).\n"
+        )
```

Comment:
This seems to be formatted funny. Could you erlfmt the file? 😅

Reply:
Done https://github.com/WhatsApp/erlfmt/pull/63/commits/eade985ed41b32fb495fa30cf636007b54e7d2a0